tests/multiprocessing_test.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import torch
+import time
+import warnings
+
+from sbi.simulators.linear_gaussian import diagonal_linear_gaussian
+from sbi.simulators.simutils import (
+    simulate_in_batches,
+    simulate_in_batches_joblib,
+)
+
+warnings.simplefilter(action="ignore", category=FutureWarning)
+
+
+def slow_linear_gaussian(theta):
+    x = []
+    for th in theta:
+        time.sleep(0.05)
+        x.append(diagonal_linear_gaussian(th.reshape(1, -1)))
+
+    return torch.cat(x)
+
+
+# @pytest.mark.parametrize("num_workers", ((5, 10)))
+# def test_benchmarking_mp(num_workers):
+
+#     num_simulations = 500
+#     theta = torch.zeros(num_simulations, 2)
+#     show_pbar = False
+#     sim_batch_size = 1
+
+#     tic = time.time()
+#     theta, x = simulate_in_batches_mp(
+#         slow_linear_gaussian,
+#         theta,
+#         sim_batch_size,
+#         num_workers,
+#         show_progressbar=show_pbar,
+#         worker_batch_size=50,
+#     )
+#     toc_mp = time.time() - tic
+#     print(f"Custom MP: {toc_mp}")
+
+#     tic = time.time()
+#     theta, x = simulate_in_batches_joblib(
+#         slow_linear_gaussian,
+#         theta,
+#         sim_batch_size,
+#         num_workers,
+#         show_progressbar=show_pbar,
+#     )
+#     toc_joblib = time.time() - tic
+
+#     print(f"JobLib: {toc_joblib}")
+
+#     # Allow joblib to be 10 percent slower.
+#     assert toc_joblib <= toc_mp * 1.1
```

Comment:
Remove comments?

Reply:
Yes, I think we can get rid of these tests anyway.